### ISSUE TITLE
[Bug]: Conflict between Access8Math and ReadFeeds

### ISSUE THREAD

**Issue by CyrilleB79** [nvdaes/readFeeds#35]
### Summary

If Access8Math  and ReadFeeds are both installed and NVDA run from source, Access8Math cannot load. See details in https://github.com/tsengwoody/Access8Math/issues/64.

### To reproduce

See https://github.com/tsengwoody/Access8Math/issues/64

### Expected behavior

Both add-ons can be loaded and work together.

### Relevant log output

```shell
ERROR - external:globalPluginHandler.listPlugins (11:22:40.640) - MainThread (17420):
Error importing global plugin Access8Math
Traceback (most recent call last):
  File "globalPluginHandler.py", line 23, in listPlugins
    plugin = importlib.import_module("globalPlugins.%s" % name, package="globalPlugins").GlobalPlugin
  File "C:\Users\Cyrille\AppData\Local\Programs\Python\Python37-32\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\__init__.py", line 99, in <module>
    from editor import EditorFrame
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\editor.py", line 9, in <module>
    from lib.viewHTML import raw2review
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\lib\viewHTML.py", line 19, in <module>
    import html5lib
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\package\html5lib\__init__.py", line 25, in <module>
    from .html5parser import HTMLParser, parse, parseFragment
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\package\html5lib\html5parser.py", line 6, in <module>
    from . import _inputstream
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\package\html5lib\_inputstream.py", line 14, in <module>
    from . import _utils
  File "C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\package\html5lib\_utils.py", line 13, in <module>
    import xml.etree.ElementTree as default_etree
ImportError: cannot import name 'etree' from 'python.xml' (C:\Users\Cyrille\AppData\Local\Temp\nvdaA8M\addons\Access8Math\globalPlugins\Access8Math\python\xml\__init__.py)
```

### Version

13.1

### System configuration

* Access8Math 3.3
* NVDA commit 3fa7c63d2afe1e59a8999b15f4e39171f543b9e3 run from source

**Comment by nvdaes** [comment]
Hi Cyrille:

I've created an elementTree branch. You can test it now or wait for me
to create a PR when I can:

https://github.com/nvdaes/readFeeds/tree/elementTree

I've imported ElementTree in functions instead of at module level.

2022-06-13 11:40 GMT+02:00, Cyrille Bougot ***@***.***>:

**Comment by nvdaes** [comment]
Seems we cannot fix this. Hope this can be fixed in NVDA itself.
